server/lib/systemd/pbench-server.service.rhel7

```diff
@@ -9,8 +9,10 @@ Documentation = https://github.com/distributed-system-analysis/pbench
 After=network.target httpd.service
 
 [Service]
+user = $(/opt/pbench-server/bin/pbench-config -l user pbench-server)
+group = $(/opt/pbench-server/bin/pbench-config -l group pbench-server)
 Type = simple
-ExecStart = /usr/bin/scl enable rh-python36 -- /opt/pbench-server/bin/pbench-server
+ExecStart = /usr/bin/scl enable rh-python36 -- gunicorn --workers 3 --bind 0.0.0.0:8001 "pbench.server.api:create_app()"
```

Comment:
Perhaps we could consider hiding all the details of how to start a `pbench-server` instance inside the `pbench-server` script itself?  Then we could have that script handle the new for using SCL vs non-SCL, and fetch configuration options for the number of workers and the host/port to bind to.

Reply:
Yes, in the recent commit I am starting the gunicorn server via pbench-server script, however I didnt notice that we now have 2 systemd service files one for rhel7 and one default. rhel7 systemd service file runs with scl and default runs without scl and the decision which one to run is made via one of the ansible roles. Does your comment implies that we get rid of 2 service file and have only 1 and decide whether to run with scl via pbench-server script?